char type
int32 minH
int32 maxH
float32 landBat
float32 param
int32 numb
